projects/cdk/directives/element/element.directive.ts

```diff
@@ -1,11 +1,17 @@
 import {Directive, ElementRef, Inject} from '@angular/core';
 
+import {tuiDownlevelElement} from './downlevel-element';
+
 @Directive({
     selector: `[tuiElement]`,
     exportAs: `elementRef`,
 })
-export class TuiElementDirective<T extends Element = HTMLElement> extends ElementRef<T> {
-    constructor(@Inject(ElementRef) {nativeElement}: ElementRef<T>) {
-        super(nativeElement);
+export class TuiElementDirective<T extends Element = HTMLElement>
+    implements ElementRef<T>
+{
+    nativeElement!: T;
+
+    constructor(@Inject(ElementRef) elementRef: ElementRef<T>) {
+        return tuiDownlevelElement(elementRef);
```

Comment:
How about we inline this return here instead, without extra function? And also add comment why we have to do this.

Reply:
fixed